samples/backups-create-with-multiple-kms-keys.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
Sample content is all good but new samples should follow [rubric](https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head#format-overview) format . Refer sample https://github.com/googleapis/nodejs-spanner/blob/main/samples/instance-partition-create.js

Reply:
Done. PTAL.